api/src/main/java/io/opentelemetry/api/trace/TraceId.java

```diff
@@ -140,6 +140,16 @@ public static String bytesToHex(byte[] traceId) {
    * @return the rightmost 8 bytes of the trace-id as a long value.
    */
   public static long getTraceIdRandomPart(CharSequence traceId) {
+    return traceIdLowBytesAsLong(traceId);
+  }
+
+  /** Convert the "high bytes" of the given hex traceId into a long representation. */
+  public static long traceIdHighBytesAsLong(CharSequence traceId) {
+    return BigendianEncoding.longFromBase16String(traceId, 0);
+  }
+
+  /** Convert the "low bytes" of the given hex traceId into a long representation. */
+  public static long traceIdLowBytesAsLong(CharSequence traceId) {
```

Comment:
Confused why use CharSequence here and String in the other place. We should be consistent.